packages/design-system-web/src/components/geov-map-circles-popup/geov-map-circles-popup.css

```diff
@@ -0,0 +1,21 @@
+:host {
+  display: block;
+  line-height: 2;
+  font-size: 1rem;
+  padding-right: 1rem;
+
+}
+
+ul {
+  list-style: none;
+  padding-left: 1rem;
+}
+
+ul ul {
+  font-size: .8rem;
+}
+
+a {
```

Comment:
I wonder why this is necessary? shouldn't we have global styles that set the styles for basics like links? or is it because of the webcomponent?